### DISCUSSION TITLE
Ch5 Probabilistic tutorial removal

### DISCUSSION THREAD

**Discussion by Martytalius** [gempy-project/gempy#594] [Q&A] [answered]
Hi Gempy Team,

I was wondering when there would be a new tutorial for probability modelling given the old tutorial has been removed from the website? I'd love to apply some uncertainty modelling to a number of personal cases.

Cheers,

Martin Nguyen

**Comment by javoha** [comment, marked as answer]
Hi @Martytalius ,

we are working on a new publication in this field and the old tutorials were deprecated. Hopefully we can provide up-to-date versions in in the not too distant future.
There is also work done in shifting the theano based backend to tensorflow, which should lead to further improvements in this direction.

**Comment by Leguark** [comment]
Yes, my personal apologies for the delay on this aspect. The development of the project has brought us into quite of a rabbit hole and the probably aspects have been parked from a while but we did not forget. Probability modeling still is at the core of `gempy`s mission.  

I am confident that during 2021 we will publish some papers and the release of `gempy v3` using TensorFlow will facilitate a lot the access to this functionality. Stay tune!
